engine/src/main/java/org/terasology/engine/core/module/ModuleManager.java

```diff
@@ -242,19 +236,4 @@ public ModuleFactory getModuleFactory() {
         return moduleFactory;
     }
 
-//    private void enrichReflectionsWithSubsystems(Module engineModule) {
-//        Serializer serializer = new XmlSerializer();
```

Comment:
why _doesn't_ the gestalt-v7 branch need an `enrichReflectionsWithSubsystems` method? v7 hasn't done away with Reflections yet…